src/config.h

```diff
@@ -96,7 +96,8 @@
 #define HAVE_ACCEPT4 1
 #endif
 
-#if (defined(__APPLE__) && defined(MAC_OS_X_VERSION_10_6)) || defined(__FreeBSD__) || defined(__OpenBSD__) || defined (__NetBSD__)
```

Comment:
maybe we can simplify the code (here and in other places) by creating some short-hand define?

e.g. MAC_OS_10_6_DETECTED ?

Reply:
MAC_OS_X_VERSION_10_6 is just the boundary point for these few code, perhaps there will be more boundary points in the future, I think it would be better to describe it directly.

Reply:
yeah, but it makes the condition complicated, so extracting it to a separate line will help, even if it's just for one condition.
also, the same one happens to be used in 3 conditions.
if we'll have more boundary points in the future, we can add more of these.

Reply:
I do agree with adding `MAC_OS_10_6_DETECTED`, I thought about it before, but I kept quiet.

Reply:
Okay.